python/cudf/cudf/tests/test_datetime.py

```diff
@@ -705,6 +705,24 @@ def test_to_datetime_units(data, unit):
         (["10/11/2012", "01/01/2010", "07/07/2016", "02/02/2014"], "%m/%d/%Y"),
         (["10/11/2012", "01/01/2010", "07/07/2016", "02/02/2014"], "%d/%m/%Y"),
         (["10/11/2012", "01/01/2010", "07/07/2016", "02/02/2014"], None),
+        (pd.Series([2015, 2020, 2021]), "%Y"),
+        pytest.param(
+            pd.Series(["1", "2", "1"]),
+            "%m",
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/6109"
+                "https://github.com/pandas-dev/pandas/issues/35934"
+            ),
+        ),
+        pytest.param(
+            pd.Series(["14", "20", "10"]),
+            "%d",
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/6109"
+                "https://github.com/pandas-dev/pandas/issues/35934"
+            ),
+        ),
+        (pd.Series([2015, 2020.0, 2021.2]), "%Y"),
```

Comment:
For us to have test coverage should we just correct the results for now and leave in a TODO comment or similar with a link to the Pandas issue?

Reply:
Was this referring to xfailed tests? We have both issues on `cudf` and `pandas` side for it so we cannot compare the correctness from `cudf` side too.

Reply:
Ah got it! Yep these tests are good to go then, resolving this.